### ISSUE TITLE
Transfer to Polyvariant

### ISSUE THREAD

**Issue by majk-p** [polyvariant/sttp-oauth2#477]
**TL;DR** the sttp-oauth2 project will be transferred to https://github.com/polyvariant/

This issue is a followup to the discussion in sttp https://github.com/softwaremill/sttp/issues/2110. It captures the steps we need to take to transfer the project.

## Transfer plan

- [x] Transfer the repository to https://github.com/polyvariant/ organization
- [x] Repackage the sources
- [x] Reconfigure deployment pipeline to publish under https://mvnrepository.com/artifact/org.polyvariant
- [x] Publish a new release with no new features first for simple migration
- [x] Update scala steward reference (https://github.com/VirtusLab/scala-steward-repos/pull/385)
- [x] Adjust mergify if necessary
- [x] Update the documentation
  - Rebrand the documentation not to use Ocado brand anymore
  - Mention the package rename in the migration guide
  - Make sure to mention Ocado as the original authors
- [x] Update the readme

Once the transfer is done we'll proceed with the waiting PRs and issues.

## Open questions
For repackaging, should we use `org.polyvariant.sttp.oauth2` or just `sttp.oauth2`?

**Comment by majk-p** [comment]
Let me know what you think @zstoychev @kubukoz

**Comment by kubukoz** [comment]
> For repackaging, should we use `org.polyvariant.sttp.oauth2` or just `sttp.oauth2`?

I don't think it's right to start with `sttp.`, so either the first one or `sttpoauth2`, that'd be my vote

**Comment by paul-sweetman-ocado** [comment]
Hi, a bit of a detail, but could you please ensure all references to Ocado are removed from the source code and documentation (except mentioning Ocado as the original author), before the repository is made public under the polyvariant org, and before any artefacts are published. Thanks.

**Comment by majk-p** [comment]
This should not be a problem, I can raise a rebranding PR first, make the last release under Ocado organization on Maven and then transfer. This should simplify migration, maybe we could even provide a scalafix rule for scala steward to automatically fix the imports :thinking:

**Comment by majk-p** [comment]
Rebranding PR is ready https://github.com/ocadotechnology/sttp-oauth2/pull/478

CC: @paul-sweetman-ocado @zstoychev

**Comment by majk-p** [comment]
The repository has now been transferred to https://github.com/polyvariant/sttp-oauth2, the last version [v0.18.0](https://github.com/polyvariant/sttp-oauth2/releases/tag/v0.18.0) has been published under Ocado organization to maven and now maven credentials are changed to the ones for Polyvariant.

The website has also been redeployed and is now published from https://sttp-oauth2.polyvariant.org

Last thing to do will be to publish v0.19.0 in the new maven org and we'll be closing this issue.

**Comment by adamw** [comment]
Great to hear :) Thanks for anybody involved

**Comment by majk-p** [comment]
[v.0.19.0](https://github.com/polyvariant/sttp-oauth2/releases/tag/v0.19.0) has just been published and should be available on maven. I think this concludes the transfer. Thanks to everyone involved!

**Comment by mzuehlke** [comment]
Adding a artefact migration rule to Scala Steward would automatically open PRs.
But the would all fail because you changed the package names. But at least it would give a hint to all users.
If you want to be fancy you could create a Scalafix rule 😀

**Comment by majk-p** [comment]
That would be neat! I considered that, but with my limited time available and ZIO support in queue I decided to deliver quickly. Hopefully the migration guide will be enough 😅

**Comment by mzuehlke** [comment]
https://github.com/scala-steward-org/scala-steward/pull/3366

**Comment by majk-p** [comment]
Thank you @mzuehlke, much appreciated!
